modules/telestion-api/src/main/java/de/wuespace/telestion/api/verticle/package-info.java

```diff
@@ -0,0 +1,27 @@
+/**
+ * <h2>Telestion API - Telestion Verticle</h2>
+ *
+ * This package contains different classes helping with Vert.X verticles by providing often used methods which help
+ * to improve the developer experience by massively reducing boiler-plate code.
+ * <h3>Usage</h3>
+ * Verticles within a Telestion Project should extend {@link de.wuespace.telestion.api.verticle.TelestionVerticle}.
+ * It provides different core functionality like more clean start() and stop() methods.
+ * <p>
+ * A Verticle usually needs a Configuration class. They must be created manually - the convention is to create a Record
```

Comment:
You could use references in all your docs. Would make refactoring much easier later on.

Reply:
Yes, this is true...